animacija2.js

```diff
@@ -56,7 +63,27 @@ function startGame() {
     drawBat()
     drawBall()
     drawBricks()
-}
+    document.body.addEventListener('keydown',function(event){
```

Comment:
Konvertzuj ovo u imenovanu funkciju. Meni ovo liči na ono što sam očekivao od batMove :)

Reply:
Radi li pomeranje?

Reply:
Radi, probaj .. ali ima jos mnogo da se kuca dok ne bude licilo na nesto

Reply:
Samo mi je sve ovo pomesano ne znam gde je sta. Bolje da smo npr uradili jednu function draw (), pa u nju pozivali samo ostale funkcije kao sto su drawBricks(), drawBat() itd. Sve te ostale bi bile van ove glavne funkcije koja se kako vec zove startGame() cini mi se.

Reply:
Da, deluje kao da je moglo i tako

Reply:
Može za prvo ispisivanje ali kasnije verovatno nećete sve vreme crtati sve, npr. dok se loptica pomera ne morate da crtate cigle non stop

Reply:
Evo u ovom linku, pravilo koje se zove currying nalaze da funkciju uvek treba razbiti na vise manjih funkcija, ako je to moguce, a ovde to svakako jeste 
https://javascript.info/currying-partials